### PR TITLE
[14.0][OU-FIX] mass_editing: magic fields sync + security group

### PR DESCRIPTION
**Pull request by pedrobaeza**
Missing pieces in the existing migration scripts:

- Transfer metadata magic fields to the created server action.
- Transfer security group (if any).

@Tecnativa TT43401

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
Thanks.

**Review by ivantodorovich — APPROVED** [review]

File: mass_editing/migrations/14.0.1.0.0/pre-migrate.py [hunk 1]
```diff
@@ -51,6 +56,10 @@ def migrate_mass_editing(env):
         )
         SELECT
             me.id,
+            create_uid,
+            create_date,
+            write_uid,
+            write_date,
```
Comment (sergiocorato) [inline mass_editing/migrations/14.0.1.0.0/pre-migrate.py:62]: I'm doing a migration and these lines throw an error (translated from italian):
_psycopg2.ProgrammingError: <ERROR:  reference to column "create_uid" is ambiguos> 
LINE 20:             create_uid,_

Comment (sergiocorato) [reply mass_editing/migrations/14.0.1.0.0/pre-migrate.py:62]:
```suggestion
            me.create_uid,
            me.create_date,
            me.write_uid,
            me.write_date,
```

Comment (legalsylvain) [reply mass_editing/migrations/14.0.1.0.0/pre-migrate.py:62]: The PR is merged. Could you make another one ?

Comment (sergiocorato) [reply mass_editing/migrations/14.0.1.0.0/pre-migrate.py:62]: Sure! #690

Comment (pedrobaeza) [reply mass_editing/migrations/14.0.1.0.0/pre-migrate.py:62]: Sorry, just the SQL I didn't check as this was amended post-mortem.

**Pull request merged**